components/pagination/assets/Pagination.scss

```diff
@@ -0,0 +1,55 @@
+@import "@not-govuk/sass-base";
+
+.hods-pagination {
+  &__summary {
+    font-family: 'Inter UI', sans-serif;
```

Comment:
Don't set the font.

Please could you apply the same feedback from #66? - Otherwise we'll just need to write all the same things.

Reply:
Yep, had already amended this before you sent that message!